chapters/1.md

```diff
@@ -3,37 +3,63 @@
+Let imagine that Leo has $2500 in his wallet in cash; Leo would be able to reach into his wallet and pull out the bills and hand them to his friend George. He could be said to be doing so *trustlessly*, because he doesn't have to trust anyone else to handle the money it is being physically handed from him to his friend and nobody can stop him from doing so.
 
-There was no real movement of physical bills that took place during the transaction. Digitally speaking, the banks edited a centralized database that now shows $2,500 being sent from Leo to George. Therein lies the problem: to establish trust between Leo and George, they had to depend on a third party: the banks. This problem is also present in the multitude of transactions happening every day, as many of those transactions cannot occur without some third party.
+But unfortunately, despite being best friends, Leo and George live on opposite sides of the country. So, even if he had cash, Leo wouldn't be able to trustlessly give the cash to his friend George because he's too far away. It's possible he could mail cash through the postal system, but then he would have to *trust* that the postal system would not lose the money, or that someone in the post office, or even a mail carrier would not steal the money.
 
-This poses a risk, because a bank (or a nefarious actor) can create an infinite amount of money by editing the transaction database or altering your own bank account - along with other illicit acts.
+Well, what if Leo and George had the same bank? Leo could instruct his bank to give the money to George via a transfer. Then, the bank would deduct the money from his account and then could add the money to George's account. In this case though, Leo has to deposit the cash he has in the bank and *trust* the bank to hold the money. 
 
-In order to help mitigate such risks, the traditional banking system can utilize a new technology called the blockchain. A blockchain is a distributed database that continuously adds to a record of all the transactions that have occurred within a network. This database is often referred to as a ledger, because the data contains a large list of transactions that have taken place.
+It's important to remember, that when you deposit money in the bank, you don't really just have money sitting in an account with your name on it. Instead, what you really have done is *lent* money to the bank. They *owe* it to you. This is what is called a *liability*. They are allowed to use your funds for other things that the bank would like to do (like making money via lending or for investments). They are required to give it back when you ask, but the money really just exists as an IOU on some database inside of their bank. So, because Leo and George have the same bank, the bank can simply change Leo's IOU to say that they owe him $2500 less, and George's IOU that they owe him $2500 more. (They debit Leo's account and subsequently credit George's account.) 
```

Comment:
What do you mean by "IOU" ? Is it "I owe you"?

Reply:
Yes. IOU is a common term in English... it's just a note that says something like "IOU $5 Mike", that I would give you so you could remember that I owe you $5. It felt awkward in the text and maybe there's a better way to write it.